src/main/java/th/ac/kmitl/science/comsci/example/controllers/DataManagement.java

```diff
@@ -0,0 +1,33 @@
+package datamanagement;
+
+import java.beans.Statement;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class DataManagement 
+{
+
+    //TaxInvoice taxInvoice = new TaxInvoice();
+    
+    public DataManagement()
+    {
+        
+    }
+    
+    public void sendTaxInvoiceToDataBase() throws ClassNotFoundException, SQLException
+    {
+        String query = "";
+        Class.forName("oracle.jdbc.driver.OracleDriver");
+        Connection connection = DriverManager.getConnection("DATABASE URL");  
+        Statement statement = (Statement) connection.createStatement();  
+        ResultSet resultSet = statement.executeQuery(query);  
```

Comment:
JDBD popular practices preferred to use `PreparedStatement` instead of raw `Statement` execution because it gives better performance and SQL Injection protection.